packages/serverpod/lib/src/server/message_central.dart

```diff
@@ -20,11 +20,17 @@ class MessageCentral {
   /// can be provided, in which case the message is sent only to that specific
   /// server within the cluster. If no [destinationServerId] is provided, the
   /// message is passed on to all servers in the cluster.
-  void postMessage(
+  ///
+  /// Returns the number of clients that received the message. Note that in a
+  /// Redis Cluster, only clients that are connected to the same node as the
+  /// publishing client are included in the count. Returns -1 if there was an
+  /// error publishing the message, or -2 if connection to the Redis server
+  /// could not be established.
+  Future<int> postMessage(
```

Comment:
Discussion: This interface might interact with other messaging services in the future. These services might not always support feedback on how many clients have received the message. That was the reason we opted for the interface not having a return value.

Do you have a specific use case for getting that information returned?

Otherwise, we could continue to keep the interface as straightforward as possible.